rdsbroker/broker.go

```diff
@@ -387,20 +391,79 @@ func (b *RDSBroker) LastOperation(instanceID string) (brokerapi.LastOperationRes
 	return lastOperationResponse, nil
 }
 
+func (b *RDSBroker) CheckAndRotateCredentials() error {
```

Comment:
It would be good to have a discussion whether we want to bundle checking and rotation of credentials in one function. Personally I would go for splitting this to two functions. `CheckCredentials` that can return list of instances that it can't connect to and `RotateCredentials` that can consume the list and change passwords. It makes those functions more generic and re-usable.